src/main/java/com/paneedah/weaponlib/WeaponRenderer.java

```diff
@@ -3001,158 +2974,137 @@ private class FirstPersonWeaponTransitionProvider implements MultipartTransition
 
 		@Override
 		public List<MultipartTransition<Part, RenderContext<RenderableState>>> getTransitions(RenderableState state) {
-			
-		
 			switch(state) {
-			case MODIFYING:
-				return getSimpleTransition(getBuilder().firstPersonPositioningModifying,
+				case MODIFYING:
```

Comment:
![image](https://github.com/Cubed-Development/Modern-Warfare-Cubed/assets/76911089/2326df7a-4321-43ce-b300-ce1ab1ecda5b)


Definitely not what I did.

GitHub having a stroke lmfao